redwood-treehouse-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -36,13 +36,19 @@ import platform.UIKit.row
 import platform.UIKit.section
 import platform.darwin.NSInteger
 
-internal class UIViewLazyColumn<A : AppService>(
+internal class UIViewLazyList<A : AppService>(
   treehouseApp: TreehouseApp<A>,
   widgetSystem: WidgetSystem,
-) : LazyColumn<UIView> {
+) : LazyList<UIView> {
   private val dataSource = TableViewDataSource(treehouseApp, widgetSystem)
   private val root = UITableView().apply {
-    this.dataSource = this@UIViewLazyColumn.dataSource
+    this.dataSource = this@UIViewLazyList.dataSource
+  }
+
+  override fun isVertical(isVertical: Boolean) {
+    if (!isVertical) {
+      TODO()
```

Comment:
@apatronl @dnagler Is it possible to toggle the scrolling direction of a `UITableView` — from vertical to horizontal? Or would we need/it be simpler to just convert over to a `UICollectionView` (which I believe supports it)?

Reply:
You'll definitely want to use a `UICollectionView` for horizontal scrolling. As per [Apple's docs](https://developer.apple.com/documentation/uikit/uitableview#//apple_ref/doc/uid/TP40006943-CH3-SW44), `UITableView`s are meant for vertical scrolling only.

Reply:
Thanks for the info! Will probably do it in a follow-up